Protein/examples/0.11-RELEASE/nextprot-jsonld.json

```diff
@@ -0,0 +1,36 @@
+{
+    "@context": "http://schema.org",
+    "@type": "DataRecord",
+    "@id": "https://www.nextprot.org/entry/NX_P52701",
+    "includedInDataset": "ftp://ftp.nextprot.org/pub/current_release/xml/nextprot_all.xml.gz",
```

Comment:
The value of this should be the value of the `@id` in your Dataset markup. It should point to the description of the dataset rather than the download file.